File: pgm/ABC_Java/ABC_418_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        reader.readLine();
        final String S = reader.readLine();
        reader.close();
        final String Yes = "Yes";
        final String No = "No";
        final String tea = "tea";
        System.out.println (S.endsWith (tea) ? Yes : No);
        System.exit (0);
    }
}

File: pgm/ABC_Kotlin/ABC_418_A.kt
// import java.time.*

// import kotlin.math.*

const val tea: String = "tea"
const val Yes: String = "Yes"
const val No: String = "No"

fun main() {
    readln()
    val S: String = readln()
    if (S.endsWith(tea)) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}